Makefile

```diff
@@ -111,6 +111,7 @@ format-cpp:
 	./bin/format $(CHECK) ./pennylane_lightning
 
 format-python:
+	isort ./pennylane_lightning/ ./mpitests ./tests $(CHECK)
```

Comment:
Using `--profile black` could help reduce redundant work done with formatting: https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
Thanks @math411 , I didn't know about this option. Sure is useful for `black` users.